### ISSUE TITLE
Compiling Problem

### ISSUE THREAD

**Issue by RickzFPV** [mlee12382/Neptune_3#2]
@mlee12382 I have a couple of Neptune 2S's as well as my Neptune 3.  All of them print really well, I want to compile some newer firmware for the 2S's but keep getting a ton of errors.  I am running the bugfix 2.0.93 now, but would like to update them to 2.0.95 or even 2.1 but nothing ever seems to lineup I get errors from the configuration h is old and some endif errors.  is it even possible to build the newer firmware for these printers.

Rick

**Comment by mlee12382** [comment]
Yeah it will work, if you change versions you need to use the matching configuration.h and configuration_adv.h files as there's often syntax changes that make copying the previous files into the new version of marlin not work. Use something like diffchecker.com to scan through and make sure you transfer the important changes from the example to the default files. Mine is running a slightly older 2.1.x bugfix. I believe the current version has changed some of the environment names so watch out for that.

**Comment by RickzFPV** [comment]
@mlee12382 how do I change the versions I have never had to do that before.  is there a version of the firmware you are running I can download I run your advanced firmware on my Neptune 3 and it works flawlessly

**Comment by mlee12382** [comment]
https://github.com/mlee12382/Elegoo-Neptune-Marlin/tree/bugfix-2.1.x/config the source code for the 2 /2S /2D is here

**Comment by RickzFPV** [comment]
Michael 1 more quick question with your configuration do I define both the 1.3 board and the 2s or just the 2s

**Comment by mlee12382** [comment]
Both, the 2S option sets the machine name and the default e-steps for the dual gear extruder.

**Comment by RickzFPV** [comment]
ok Perfect
